src/mongo/db/auth/auth_index_d.cpp

```diff
@@ -100,7 +100,7 @@ Status verifySystemIndexes(OperationContext* txn) {
     std::vector<IndexDescriptor*> indexes;
     indexCatalog->findIndexesByKeyPattern(txn, v1SystemUsersKeyPattern, false, &indexes);
 
-    if (indexCatalog && !indexes.empty()) {
+    if (!indexes.empty()) {
```

Comment:
'indexCatalog' is assigned a pointer to a field of object 'collection' of class 'Collection', so it cannot be null.